src/Core/Silk.NET.Core/Native/Structs/FileTime.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silk.NET.Core.Native
+{
+    public struct FileTime
```

Comment:
This struct already exists in Silk.NET.Core.Win32Extras

Reply:
I missed that, it does mean that the IStream would need to move to that as they require this.

Reply:
this is now fixed as objidlbase.h was added to win32extras